cmd/scanip.go

```diff
@@ -47,7 +47,7 @@ func ScanIP(hostUrl string, serverUrl string, wg *sync.WaitGroup, resChan chan s
 		log.Fatal(err)
 	}
 	request.Header.Set("User-Agent", targetUserAgent)
```

Comment:
why not move this header into the list as well?

Reply:
@guybarnhartmagen  since we are adding different value here.

should I change it to:
```
func addCommonHeaders(headers *http.Header, headerValue string, userAgentValue string) {
    for _, header := range commonHeaders {
        if header == "User-Agent" {
            headers.Set(header, userAgentValue)
        }
        headers.Add(header,headerValue)
    }
}
```

please be more explicit on what we need here.